dashboard/src/reducers/auth.ts

```diff
@@ -6,25 +6,39 @@ import { AuthAction } from "../actions/auth";
 export interface IAuthState {
   authenticated: boolean;
   authenticating: boolean;
+  checkingOIDCToken: boolean;
+  autoAuthenticated: boolean;
```

Comment:
`autoAuthenticated` could be `oidc: boolean` or `oidcAuthenticated: boolean` to make it more clear what the intended usecase is